### PR TITLE
Added support for KrakenZ Devices

### PR DESCRIPTION
**Pull request by sepp89117**
Added support for KrakenZ Devices with pump and fan control. The LCD is not treated.

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    private static void FillTargetArray(byte[] targetArray, byte value)
+    {
+        for (byte i = 4; i < targetArray.Length; i++)
+            targetArray[i] = value;
+    }
+
+    static KrakenZ()
+    {
+        // Init pump control target array
+        Array.Clear(_setPumpTarget, 0, _setPumpTarget.Length);
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs:40]: This is useless.

File: LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+     */
+internal sealed class KrakenZ : Hardware
+{
+    private static readonly byte[] _getFirmwareInfo = { 0x10, 0x01 };
+    private static readonly byte[] _setPumpSpeedHeader = { 0x72, 0x01, 0x00, 0x00 };
+    private static readonly byte[] _setFanSpeedHeader = { 0x72, 0x02, 0x00, 0x00 };
+    private static readonly byte[] _status_req = { 0x74, 0x01 };
+
+    private static readonly byte[] _setPumpTarget = new byte[64];
+    private static readonly byte[] _setFanTarget = new byte[64];
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs:19]: Sort / format document.

File: LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+    static KrakenZ()
+    {
+        // Init pump control target array
+        Array.Clear(_setPumpTarget, 0, _setPumpTarget.Length);
+        Array.Copy(_setPumpSpeedHeader, 0, _setPumpTarget, 0, 4);
+        // Fill target array with a pump duty of 60%
+        FillTargetArray(_setPumpTarget, 60);
+
+        // Init fan control target array
+        Array.Clear(_setFanTarget, 0, _setFanTarget.Length);
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenZ.cs:46]: Same.

**Comment by sepp89117** [comment]
Sorry, I'm just a hobby programmer.
I changed what I recognized as useless or to format.
I can't do more without more detailed instructions.

**Pull request merged**